src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -5475,6 +5475,163 @@
                 "path": "https://creativecommons.org/licenses/by/4.0/"
             }],
             "sources": []
+        },
+        {
+            "profile": "tabular-data-resource",
+            "name": "plant_unit_epa",
+            "path": "data/assn_eia_epacems.csv",
```

Comment:
Is it okay that the path and the name are different here? I had thought that they needed to be the same.

Reply:
Hmmm, I'm not sure about the "legality" of having different names path/table names, but in this case they are difference because we are taking one CSV and splitting it into three separate normalized tables.

@zaneselvans do you think this will be an issue?

Reply:
oooh i think i understand why this isn't a problem rn. Because we are always overwritting the path when we generate the "real" datapackage metadata in `pudl.load.metadata.get_tabular_data_resource()`.

Reply:
is the path supposed to be the path to the raw datafile or the output datafile?

Reply:
the output file! which is why it doesn't have an effect in this file because this is the thing that we use to cobble together the metadata for the datapackage once it is generated.